### PR TITLE
Story/1670 Branding - Removing Odoo References

### PR DESCRIPTION
**Pull request by jrbirch99**
Removed references to Odoo and reworded some text.

### CODE REVIEW

**Review by PeterAlabaster — APPROVED** [review]

File: addons/mail/data/mail_data.xml [hunk 1]
```diff
@@ -4,7 +4,6 @@
 
         <record id="base.partner_root" model="res.partner">
             <field name="name">OdooBot</field>
```
Comment (PeterAlabaster) [inline addons/mail/data/mail_data.xml:6]: Is it in scope to change this also from `OdooBot`?

File: odoo/addons/base/views/res_partner_views.xml [hunk 2]
```diff
@@ -208,7 +208,7 @@
                             <field name="mobile" widget="phone"/>
                             <field name="user_ids" invisible="1"/>
                             <field name="email" widget="email" context="{'gravatar_image': True}" attrs="{'required': [('user_ids','!=', [])]}"/>
-                            <field name="website" widget="url" placeholder="e.g. https://www.odoo.com"/>
```
Comment (lcucurachi) [inline odoo/addons/base/views/res_partner_views.xml]: Wouldn't it be better to replace it with https://www.example.com ? Or does PO want to remove them completely? Same above

File: addons/mail/data/mail_data.xml (hunk 1 above)
Comment (jrbirch99) [reply addons/mail/data/mail_data.xml:6]: This already get overwritten in UDES branding to UDES bot. I found in the UI when you click on the UDES bot profile, there were log messages at the bottom of the screen showing how the email changed (as the various different models were installed) "e.g. root@example.com -> odoobot@example.com -> udesbot@example.com", so I removed the email here to stop reference appearing.

**Comment by armandcela** [comment]
General Comment: I would say maybe if we would inherit these changes in udes module not changing odoo itself :)
But that may be skipped as we have created a branch out of odoo and there wont be cases where we need to update odoo branch

File: odoo/addons/base/views/res_partner_views.xml (hunk 2 above)
Comment (jrbirch99) [reply odoo/addons/base/views/res_partner_views.xml]: The get set to "https://www.unipart.com", after udes_branding is installed. https://gitlab.unipart.digital/udes/udes_closed_14/-/merge_requests/104/diffs?diff_head=true#:~:text=%3Ee.g.%20https%3A%2F%2Fwww.unipart.com%3Cattribute%2F%3E

**Review by lcucurachi — APPROVED** [review]

**Comment by jrbirch99** [comment]
> General Comment: I would say maybe if we would inherit these changes in uded module not changing odoo itself :) But that may be skipped as we have created a branch out of odoo and there wont be cases where we need to update odoo branch

I did think about inheriting all these changes in udes_branding. I guess the changes I have made are not strictly branding for udes, but removing references to odoo hence not sure if they would live in udes_branding. We have changed the odoo module in the past, to remove odoo references (with the odoo references in error message popups) so I thought it was best to be consistent with that.

**Review by armandcela — APPROVED** [review]

**Comment by lcucurachi** [comment]
> > General Comment: I would say maybe if we would inherit these changes in uded module not changing odoo itself :) But that may be skipped as we have created a branch out of odoo and there wont be cases where we need to update odoo branch
> 
> I did think about inheriting all these changes in udes_branding. I guess the changes I have made are not strictly branding for udes, but removing references to odoo hence not sure if they would live in udes_branding. We have changed the odoo module in the past, to remove odoo references (with the odoo references in error message popups) so I thought it was best to be consistent with that.

The fact is that we have the Odoo branch because most likely we want to be able to modify Odoo if we need to, like in this case.

The approach Armand is suggesting is not bad or wrong at all.

Personally, I think the reason why it was decided to have an Odoo repo is that the TLs wanted to have that ability to change Odoo directly in one place if we needed to. Also if we have the Odoo repo we can run Odoo tests if we want to which is really useful to see if we are breaking anything from Odoo so actually I like the fact that we have it in the first instance. It has some advantages in my opinion.

TLs might be able to say more in this case. It is an interesting question to ask why one approach was considered instead of the other.

File: odoo/models.py [hunk 3]
```diff
@@ -6421,9 +6421,9 @@ def __len__(self):
 AbstractModel = BaseModel
 
 class Model(AbstractModel):
-    """ Main super-class for regular database-persisted Odoo models.
+    """ Main super-class for regular database-persisted models.
 
-    Odoo models are created by inheriting from this class::
+    Models are created by inheriting from this class::
```
Comment (caleb-shelton) [inline odoo/models.py]: Do we really need to scrub docstrings?

File: addons/digest/data/digest_tips_data.xml [hunk 4]
```diff
@@ -2,7 +2,7 @@
 <odoo>
     <data noupdate="1">
         <record id="digest_digest_default" model="digest.digest">
-            <field name="name">Your Odoo Periodic Digest</field>
+            <field name="name">Your Periodic Digest</field>
```
Comment (caleb-shelton) [inline addons/digest/data/digest_tips_data.xml]: Are we actually using the digest addon?

Comment (jrbirch99) [reply addons/digest/data/digest_tips_data.xml]: yep, when running UDES gentoo its appears in the settings menu.

File: odoo/models.py (hunk 3 above)
Comment (jrbirch99) [reply odoo/models.py]: So James found an instance where this is displayed in a notes section of a record. I went to double check and found that it is only visible in debug mode, so I don't think a user will ever be able to see it. I will undo these changes.

**Review by caleb-shelton — APPROVED** [review]
LGTM

**Pull request merged**